roles/oauth/templates/config.json.j2

```diff
@@ -285,6 +285,15 @@
       "redirectUri": "https://example2.domain/return?foo=bar",
       "trusted": true,
       "canGrant": true
+    },
+    {
+      "id": "1b1a3e44c54fbb58",
+      "hashedSecret": "894669f8a640e824a52e268d51d7732b1b77267300a11cc77145ad0ef81a0c17",
```

Comment:
@vbudhram can you paste the secret into this issue, so we remember it

Reply:
This is a public client (like desktop firefox) so it shouldn't ever need to know the corresponding secret.

Reply:
Ah, so no use for the secret at all?  ok 👍